ensembl_dbcopy/models.py

```diff
@@ -160,8 +165,10 @@ def table_status(self):
                 return 'Failed'
             elif self.job_id.status == 'Creating Requests':
                 return 'Submitted'
-            else:
+            elif self.job_id.status == 'Processing Requests':
                 return 'Running'
+            else:
+                return 'Submitted'
         else:
             return 'Submitted'
```

Comment:
Similarly here, can we simplify with only one `return 'Submitted'`?

Reply:
Makes perfect sense thanks!